pkg/apis/crd/v1alpha1/types.go

```diff
@@ -354,3 +355,148 @@ type BGPPeer struct {
 	// a restart before deleting stale routes. The range of the value is from 1 to 3600, and the default value is 120.
 	GracefulRestartTimeSeconds *int32 `json:"gracefulRestartTimeSeconds,omitempty"`
 }
+
+type PodReference struct {
+	Namespace string `json:"namespace"`
+	Name      string `json:"name"`
+}
+
+// Source describes the source spec of the packetcapture.
+type Source struct {
+	// Pod is the source Pod, exclusive with source `IP`.
```

Comment:
```suggestion
	// Pod is the source Pod, mutually exclusive with IP.
```

not sure why you used backticks specifically here

Reply:
updated.